src/pudl/transform/eia860.py

```diff
@@ -356,6 +356,70 @@ def _core_eia860__generators(
     return gens_df
 
 
+@asset
+def _core_eia860__generators_energy_storage(
+    raw_eia860__generator_energy_storage_existing: pd.DataFrame,
+    raw_eia860__generator_energy_storage_retired: pd.DataFrame,
+) -> pd.DataFrame:
+    """Transform the energy storage specific generators table.
+
+    Notes for possible future cleaning:
+    * A lot of the values that are repeated in the generators table are dropped post
+    harvesting. It would be nice to compare the values before dropping them to make sure
+    these specific generator tables don't contain information that the big generator
+    tables does not.
+    """
+    storage_ex = raw_eia860__generator_energy_storage_existing
+    storage_re = raw_eia860__generator_energy_storage_retired
+
+    boolean_columns_to_fix = [
+        "served_arbitrage_applications",
+        "served_backup_power_applications",
+        "served_co_located_renewable_firming",
+        "served_frequency_regulation_applications",
+        "served_load_following_applications",
+        "served_load_management_applications",
+        "served_ramping_spinning_reserve_applications",
+        "served_system_peak_shaving_applications",
+        "served_transmission_and_distribution_deferral_applications",
+        "served_voltage_or_reactive_power_support_applications",
+        "stored_excess_wind_and_solar_generation",
+    ]
```

Comment:
I was torn about this because I sort of thought the opposite - that if a column name changed it would be easier to flag if the column name was fully written out (i.e. searchable in the code base) as opposed to found via `filter` and prefix codes. But I don't feel super strongly here. Better to be consistent.

Reply:
It's definitely prettier not to have them written out